feature.env

```diff
@@ -0,0 +1,12 @@
+PORT=3006
+OAUTH_ENDPOINT_URL=http://localhost:19090/auth/
```

Comment:
feature config in package.json was getting too big, so moved to separate env file